README.md

```diff
@@ -489,16 +489,17 @@ Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`
 
 The table below shows the supported package managers and their support level in Cachito.
 
-Feature                 | gomod | npm | pip |
----                     | ---   | --- | --- |
-Baseline                | ✓     | ✓   | ✓   |
-Content Manifest        | ✓     | ✓   | ✓   |
-Dependency Replacements | ✓     | x   | x   |
-Dev Dependencies        | ✓     | ✓   | ✓   |
-External Dependencies   | N/A   | ✓   | ✓   |
-Multiple Paths          | x     | ✓   | ✓   |
-Nested Dependencies     | ✓     | ✓   | x   |
-Offline Installations   | ✓     | x   | x   |
+Feature                 | gomod | npm | pip | git-submodule |
+---                     | ---   | --- | --- |               |
+Baseline                | ✓     | ✓   | ✓   | ✓⁺            |
+Content Manifest        | ✓     | ✓   | ✓   | ✓             |
+Dependency Replacements | ✓     | x   | x   | x             |
+Dev Dependencies        | ✓     | ✓   | ✓   | x             |
+External Dependencies   | N/A   | ✓   | ✓   | N/A           |
+Multiple Paths          | x     | ✓   | ✓   | x             |
```

Comment:
We do support multiple paths in the sense that if the repo contains multiple submodules, we will fetch them all. It is just not configurable. Even if we do drop `git-submodule` from this table, it might be worth mentioning elsewhere.